validation/src/test/scala/hmda/validation/rules/lar/macro/lessThanOrEqualToPropertyMacroSpec.scala

```diff
@@ -0,0 +1,37 @@
+package hmda.validation.rules.lar.`macro`
+
+import com.typesafe.config.ConfigFactory
+import hmda.model.fi.lar.LoanApplicationRegister
+
+trait lessThanOrEqualToPropertyMacroSpec extends MacroSpec {
+
```

Comment:
This goes against naming conventions, the name of a `trait` is usually capitalized